### PR TITLE
Add B & N data scraper

### PR DESCRIPTION
**Pull request by mperez14**
Made changes.
Fixed comments,
Fixed old code returning error on false and true ISBN's (reason was that inputting ISBN's doesn't take you to a page with results loaded, it takes you directly to the corresponding book's page) Code now returns everything

### CODE REVIEW

File: scraper.py [hunk 1]
```diff
@@ -171,3 +171,51 @@ def get_book_type(item):
...
+
+        new_PurchaseOption.seller = 'Barnes and Noble'  #seller
+
+        return new_PurchaseOption
+    else:
+        print "No results found at 'http://www.barnesandnoble.com' for '%s'" %keyword
+
+
+
+def get_page_for_Chegg_book_search(keyword):
```
Comment (cshinaver) [inline scraper.py]: I saw you removed the other Chegg methods. Is there a reason why you kept this one?

File: scraper.py [hunk 2]
```diff
@@ -171,3 +171,51 @@ def get_book_type(item):
...
+        new_PurchaseOption = PurchaseOption()
+        list_wrapper_item = soup.find('section', id='prodSummary')
+        product_info = list_wrapper_item.find('li', class_='tab selected')
+
+        item_url_extension = (product_info.find_all('a')[0]).attrs['href']
+        item_base = "http://www.barnesandnoble.com"
+        item_url = item_base+item_url_extension
+        new_PurchaseOption.link = item_url          #get link
+
+        new_PurchaseOption.is_rental = False #isRental
```
Comment (cshinaver) [inline scraper.py]: Were you not able to handle the books that were available for rental?

**Comment by mperez14** [comment]
With previous master merged

**Comment by cshinaver** [comment]
Looks good

**Pull request merged**